### ISSUE TITLE
Discussion: Multiple labels in a multiline environment

### ISSUE THREAD

**Issue by peterzjx** [tomduck/pandoc-eqnos#59]
Similar to https://github.com/lierdakil/pandoc-crossref/issues/184 , I was looking for a solution to support numbering and referencing multiple labels within one multiline equation. 

My intended target format is html5+mathjax only, so I am able to use some features provided by mathjax. I made a fork here https://github.com/peterzjx/pandoc-eqnos to demonstrate some basic ideas if anyone is interested.
